src/js/components/Button/index.d.ts

```diff
@@ -27,7 +27,9 @@ export interface ButtonProps {
   href?: string;
   target?: "_self" | "_blank" | "_parent" | "_top";
   icon?: JSX.Element;
+  justify?: "around" | "between" | "center" | "end" | "evenly" | "start" | "stretch";
   label?: React.ReactNode;
+  normalized?: boolean;
```

Comment:
`simple`

Reply:
Thank you! Committed these changes yesterday, but forgot to push!